apps/Electron Fiddle/install-32

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+fiddle_ver="electron-fiddle_0.27.0_armhf.deb"
+install_packages https://github.com/electron/fiddle/releases/download/v0.27.0/electron-fiddle_0.27.0_armhf.deb
+rm -rf ${fiddle_ver} || sudo rm -rf ${fiddle_ver}
```

Comment:
```suggestion
rm -rf ${fiddle_ver} || sudo rm -rf ${fiddle_ver}
```
`install_packages` already handles that.

Reply:
oh, ok.